gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.2-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.4.2-bin.zip
```

Comment:
Can you please use `all` here.
Also can you upgrade version with `./gradlew wrapper --gradle-version=7.4.2 --distribution-type=all` (in case if you did not) so other related files can also be updated.

Reply:
Done